app/src/main/java/com/yeonae/chamelezone/view/place/adapter/PlaceReviewTabRvAdapter.kt

```diff
@@ -1,52 +1,79 @@
 package com.yeonae.chamelezone.view.place.adapter
 
+import android.content.Intent
 import android.view.LayoutInflater
+import android.view.View
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import com.yeonae.chamelezone.R
-import com.yeonae.chamelezone.data.model.Like
-import kotlinx.android.synthetic.main.item_like.view.*
+import com.yeonae.chamelezone.data.model.Review
+import com.yeonae.chamelezone.view.review.MyReviewDetailActivity
+import kotlinx.android.synthetic.main.item_place_review.view.*
 
-class PlaceReviewTabRvAdapter : RecyclerView.Adapter<PlaceReviewTabRvAdapter.PlaceReviewViewHolder>() {
+class PlaceReviewTabRvAdapter(private val reviewList: ArrayList<Review>) :
+    RecyclerView.Adapter<PlaceReviewTabRvAdapter.PlaceReviewViewHolder>() {
 
-    private var items = mutableListOf<Like>()
-    private var onClickListener: OnClickListener? = null
+//    private var onClickListener: OnClickListener? = null
 
-    interface OnClickListener {
-        fun onClick(productResponse: Like)
+    interface OnItemClickListener {
+        fun onItemClick(view: View, position: Int)
     }
 
-    fun setOnClickListener(listener: OnClickListener) {
-        onClickListener = listener
+
+    class PlaceReviewViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        private val placeReviewList: ArrayList<Review>? = null
+        private val itemClickListener: OnItemClickListener? = null
+        private val userId = itemView.tv_user_id
```

Comment:
```suggestion
        private val userId: TextView = itemView.findViewById(R.id.tv_user_id)
```
viewholder에서 kotlin-extension 을 사용하면 성능상의 문제가 있다고합니다.

Reply:
수정했습니다!
ba7265e